### ISSUE TITLE
Different formatting in iOS vc Android

### ISSUE THREAD

**Issue by egorges** [daohoangson/flutter_widget_from_html#186]
Looks like standard formatting looks different from iOS and Android, see below.

My Theme:     
```
static const _defaultSize = 20.0;  //body1
ThemeData(
      primaryColor: K.color_main, //Colors.green,
      accentColor: K.color_highlight, //Colors.green[500],
      secondaryHeaderColor: K.color_background,
      fontFamily: K.regularFont,
      textTheme: TextTheme(
        headline: TextStyle(fontSize: 36.0, fontWeight: FontWeight.w300,   fontFamily: K.titleFont),
        title:    TextStyle(fontSize: _defaultSize *1.2), 
        subtitle: TextStyle(fontSize: _defaultSize, color: K.color_disabled),
        subhead:  TextStyle(fontSize: _defaultSize), 
        caption:  TextStyle(fontSize: 8.0),
        button:   TextStyle(fontSize: 12.0),
        overline: TextStyle(fontSize: _defaultSize, color:K.color_background),
        body1:    TextStyle(fontSize: _defaultSize),
        body2:    TextStyle(fontSize: _defaultSize *0.7),
        display1: TextStyle(fontSize: _defaultSize, color:K.color_acertou),
        display2: TextStyle(fontSize: _defaultSize, color:K.color_errou),
        display3: TextStyle(fontSize: _defaultSize, color:K.color_outra),
        display4: TextStyle(fontSize: _defaultSize, color:K.color_emAberto),

```
but  HtmlWidget(text, textStyle: Theme.of(context).textTheme.**body1**.copyWith(fontFamily: K.questionsFont)) gives me: 

on Android:
![Screenshot_1589840758](https://user-images.githubusercontent.com/47680522/82265511-82848880-993d-11ea-83e2-d255b04a69e7.png)

on IOS:
![Simulator Screen Shot - iPhone 8 - 2020-05-18 at 19 23 16](https://user-images.githubusercontent.com/47680522/82265350-26216900-993d-11ea-861f-88b16b036342.png)

Although **display3** (used on the five answers below) looks ok:

on Android:
![Screenshot_1589840982](https://user-images.githubusercontent.com/47680522/82265701-0179c100-993e-11ea-8cf6-75d093ec0258.png)

on IOS:
![Simulator Screen Shot - iPhone 8 - 2020-05-18 at 19 29 31](https://user-images.githubusercontent.com/47680522/82265685-f757c280-993d-11ea-8a7b-dc2d003dd55e.png)

**Comment by daohoangson** [comment]
Can you try with a different font family? Or use the system font? In the iOS screenshot, the `ç` character is smaller than other characters. I think this maybe a font issue.

Also, there are 2 things that effect how big the text is: font size and text scale. Normally the text scale is 1 but it will be bigger if you turn on system accessibility mode (e.g. larger text). Can you check this on the iOS device / simulator to make sure it's off?

**Comment by egorges** [comment]
> Can you try with a different font family? Or use the system font? In the iOS screenshot, the `ç` character is smaller than other characters. I think this maybe a font issue.

I am playing with different fonts to find a workaround.

> Also, there are 2 things that effect how big the text is: font size and text scale. Normally the text scale is 1 but it will be bigger if you turn on system accessibility mode (e.g. larger text). Can you check this on the iOS device / simulator to make sure it's off?

I checked it's off. 

Not sure why the answers format the same size (based on display3) but the text is different with body1.

**Comment by daohoangson** [comment]
> I am playing with different fonts to find a workaround.

Does it work with a different font?

> I checked it's off. Not sure why the answers format the same size (based on display3) but the text is different with body1.

Yeah, that's weird...

**Comment by egorges** [comment]
> Does it work with a different font?
> 
> >

Not perfectly.
The best workaround is to use display3 instead of body1.

**Comment by daohoangson** [comment]
Can you try setting the font size and family directly?

```dart
HtmlWidget(
  text,
  textStyle: TextStyle(
    fontSize: _defaultSize,
    tontFamily: K.questionsFont,
  )
)
```

I think the theme data is inherit internally and somehow it changed the font size.

**Comment by egorges** [comment]
@daohoangson , I couldn't retest it and just figured out I overwritten the font file with a new one. 

I think it is related to the font I used has standard big characters.

For the record, most of my characters came from Roboto, ASANA Math and XITS Math.

**Comment by daohoangson** [comment]
Ah, so it is a font issue?

**Comment by egorges** [comment]
I guess so. I'll play around with it a little more to see if I find the issue.

Maybe some font "general settings" I didn't intend to change was changed.

**Comment by egorges** [comment]
@daohoangson , I figured out what was the problem.

I used FontForge to merge characters from different fonts, and then I generate a new one.

The correct way to generate the new is is with the Apple flag disabled (besides default is enabled) and OpenType enabled:
![FontForge-opentype-options-window](https://user-images.githubusercontent.com/47680522/82707910-64779a80-9c53-11ea-8bf2-f82ab3a4c725.png)
Source: http://designwithfontforge.com/en-US/The_Final_Output_Generating_Font_Files.html

Though this is not a plugin problem, I'm closing the issue. Thanks
